.github/workflows/release-container.yaml

```diff
@@ -0,0 +1,66 @@
+name: Release Container Image
+
+on:
+  push:
+    tags:
+      - 'wallet-v[0-9]+.[0-9]+.[0-9]+' # support v1.2.3
+      - 'wallet-v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+' #supports release-candidate versions v1.2.3-rc.4
```

Comment:
This workflow is for releasing a container image, but we also need a workflow to be used in e.g. pull requests that tries to build the code and run the tests.